File: java-tdd-template/src/test/java/com/connexxo/tdd_template/Class1Tests.java
package com.connexxo.tdd_template;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class Class1Tests {

    @Test void FailingTest() {
        Class1 underTest = new Class1();
        assertTrue(false, "I'm a failing test!");
    }

    @Test void PassingTest() {
        assertTrue(true, "I'm a passing test!");
    }
}

File: csharp-tdd-template/TddTemplate.Tests/Class1Tests.cs
using Xunit;

namespace TddTemplate.Tests
{
    public class Class1Tests
    {
        [Fact]
        public void FailingTest()
        {
            var underTest = new Class1();
            Assert.True(false, "I'm a failing test!");
        }

        [Fact]
        public void PassingTest()
        {
            Assert.True(true, "I'm a passing test!");
        }
    }
}